src/libpgmoneta/utils.c

```diff
@@ -79,6 +79,8 @@ static int get_permissions(char* from, int* permissions);
 
 static void copy_file(void* arg);
 
+char* excluded_files_names[] = {"/global/pg_control"};
```

Comment:
Make this an API call such that it can be defined in `restore.c`

Reply:
done and moved to `restore.c`